sql/executor.go

```diff
@@ -206,8 +206,10 @@ func (e Executor) execStmt(stmt parser.Statement, params parameters, planMaker *
 						row.Values = append(row.Values, driver.Datum{
 							Payload: &driver.Datum_IntervalVal{IntervalVal: vt.Nanoseconds()},
 						})
+					case parser.DTuple:
+						return fmt.Errorf("unsupported result type: tuple, remove extra parenthesis around column %s", result.Columns[i])
 					default:
-						return util.Errorf("unsupported datum: %T", val)
+						return util.Errorf("unsupported result type: %T", val)
```

Comment:
Well, yes, because it prints a go type rather than a sql type. This should probably also change from `util.Errorf` to `fmt.Errorf` if it's to be a user-facing error message.